products/sle/main.pm

```diff
@@ -806,6 +806,9 @@ elsif (get_var("FIPS_TS") || get_var("SECURITY")) {
     elsif (check_var("SECURITY", "apparmor_status")) {
         load_security_tests_apparmor_status;
     }
+    elsif (check_var("SECURITY", "apparmor_genprof")) {
+        load_security_tests_apparmor_genprof;
+    }
```

Comment:
Now I see that the above block in both the opensuse/main.pm as well as sle/main.pm are very common. Could you please try to also 
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html). That could be something for a followup PR of course as well so feel free to tell me that you do not want to change this now but later

Reply:
Good suggestion. I think I should do it. But since it is related to overall "if...elseif" code block, I need to think about how to involve FIPS var to this change. I tend to create a new poo and a separate PR in a later time.